core/templates/dev/head/components/OutcomeFeedbackEditorDirective.js

```diff
@@ -33,4 +33,5 @@ oppia.directive('outcomeFeedbackEditor', [
         };
       }]
     };
-  }]);
+  }
+]);
```

Comment:
Any specific reason for modifying this? If not, please change this back and club the brackets together.

Reply:
Done..!